src/index.js

```diff
@@ -33,19 +33,24 @@ app.use(async (req, res, next) => {
     switch (type) {
       case 'pdf':
         const urlObj = new URL(url)
-        let filename = urlObj.hostname
-        if (urlObj.pathname !== '/') {
-          filename = urlObj.pathname.split('/').pop()
-          if (filename === '') filename = urlObj.pathname.replace(/\//g, '')
-          const extDotPosition = filename.lastIndexOf('.')
-          if (extDotPosition > 0) filename = filename.substring(0, extDotPosition)
+        if (!filename || String(filename).length === 0) {
```

Comment:
This code could be replaced with ```if (!filename) {```

Reply:
`!''` evalues to `true`. `.pdf` wouldn't be a good name, right?

Reply:
If `filename` parameter receive empty string, filename would be `urlObj.hostname`.pdf.

Reply:
Sorry, you are right.

Reply:
Updated.